lib/Listeners/LoadAdditionalScripts.php

```diff
@@ -44,7 +44,7 @@ public function handle(Event $event): void {
 			return;
 		}
 
-		Util::addScript(Application::APP_ID, 'files');
+		Util::addInitScript(Application::APP_ID, 'files_lock-main');
```

Comment:
```suggestion
		Util::addInitScript(Application::APP_ID, 'files_lock-init');
```

Reply:
Done, split to separate init and main file to only perform necessary tasks early